coala-decorators/Decorators.py

```diff
@@ -0,0 +1,326 @@
+import inspect
```

Comment:
i moved it to a different folder, to package everything. i dont know why but it seems that i need to have the files that i want packaged in a different folder that is placed where the readme, license, setups are.